docs/iam-usage.rst

```diff
@@ -0,0 +1,148 @@
+Using IAM For Users
+===================
+
+.. note:: This document assumes basic knowledge of Google Cloud IAM,
+   see `the docs <https://cloud.google.com/iam/docs/>`_ for details
```

Comment:
No need to change, but inline links are harder to use since they take up lots of space. For future ref you can also do

```
`the docs`_ for details

.. _the docs: https://cloud.google.com/iam/docs/
```